src/components/DocumentationTopic.vue

```diff
@@ -300,6 +301,22 @@ export default {
     ),
     shouldShowLanguageSwitcher: ({ objcPath, swiftPath }) => objcPath && swiftPath,
     hideSummary: () => getSetting(['features', 'docs', 'summary', 'hide'], false),
+    enableHero: ({ symbolKind }) => (symbolKind ? (symbolKind === 'module') : true),
+    heroTitleStyle({ enableHero }) {
+      return enableHero ? {
```

Comment:
Generally I would suggest we apply such css vars with JS logic, only in cases where we need a dynamic value from the JS env itself.

In this case it might make sense to just toggle a class on/off and apply these in pure the SCSS code.
Also `light-color` is a scss function, it should not work here.